### PR TITLE
Feature/setcustomvar example

### PR DESCRIPTION
**Pull request by 9renpoto**
Connect to https://github.com/uncovertruth/docs.userdive.com/pull/87

Update customvar examples

### CODE REVIEW

File: src/t/setcustomvar.pug [hunk 1]
```diff
@@ -2,11 +2,37 @@ extends ../integration/index.pug
 
 block footer_content
   script.
-    ud("create", "10-732", { "cookieExpires": 1});
+    /**
+     * set customvar before analytics start
```
Comment (9renpoto) [inline src/t/setcustomvar.pug]: please review comment!

File: src/t/setcustomvar.pug [hunk 2]
```diff
@@ -2,11 +2,37 @@ extends ../integration/index.pug
...
     ud("analyze");
   script.
-    var t1 = 'customVar1';
-    var t2 = 'customVar2 sample';
-    ud('setCustomVar', [
-      t1, t2, "SampleVar3"
-    ]);
+    setTimeout(function () {
+      /**
+       * set customvar when wait a 5 min
```
Comment (9renpoto) [inline src/t/setcustomvar.pug]: please review comment!

File: src/t/setcustomvar.pug [hunk 3]
```diff
@@ -2,11 +2,37 @@ extends ../integration/index.pug
...
+       * @param  {Array} Update customvar, customVar1: "5 minites"
+       * @return {void}
+       */
+      ud('setCustomVar', ["5 minitues"]);
+    }, 1000 * 60 * 5)
+
+  script.
+    $("#about").on("click", function() {
+      /**
+       * set customvar when #about click
```
Comment (9renpoto) [inline src/t/setcustomvar.pug]: please review comment

File: src/t/setcustomvar.pug (hunk 1 above)
Comment (Surgo) [inline src/t/setcustomvar.pug]: "set custom variables before starting a tracking" かな

File: src/t/setcustomvar.pug [hunk 4]
```diff
@@ -2,11 +2,37 @@ extends ../integration/index.pug
 
 block footer_content
   script.
-    ud("create", "10-732", { "cookieExpires": 1});
+    /**
+     * set customvar before analytics start
+     * @param  {string} "create" api key name, initialize Tracker
```
Comment (Surgo) [inline src/t/setcustomvar.pug]: "create" is function name, which initialize the Tracker かな

File: src/t/setcustomvar.pug [hunk 5]
```diff
@@ -2,11 +2,37 @@ extends ../integration/index.pug
...
+     * @param  {string} "10-732" ProjectId
+     * @param  {Object} {"customVar": ["init"]}  set first customvar, customVar1: "init"
```
Comment (Surgo) [inline src/t/setcustomvar.pug]: `set a first custom variable, e.g,. "init" is setted to customVar1`

File: src/t/setcustomvar.pug (hunk 2 above)
Comment (Surgo) [inline src/t/setcustomvar.pug]: set custom variables after 5 minutes

File: src/t/setcustomvar.pug (hunk 3 above)
Comment (Surgo) [inline src/t/setcustomvar.pug]: set custom variables when clicking #about

File: src/t/setcustomvar.pug (hunk 5 above)
Comment (tsuyukimakoto) [inline src/t/setcustomvar.pug]: setはset,set,setではなかったか？

Comment (9renpoto) [inline src/t/setcustomvar.pug]: set set set とは

Comment (tsuyukimakoto) [inline src/t/setcustomvar.pug]: なんていうんでしたっけ？現在、過去、かこぶんしが同じ形のやつ

File: src/t/setcustomvar.pug [hunk 6]
```diff
@@ -2,11 +2,37 @@ extends ../integration/index.pug
 
 block footer_content
   script.
-    ud("create", "10-732", { "cookieExpires": 1});
+    /**
+     * set custom variables before starting a track
+     * @param  {string} "create" is function name, which initialize the Tracker
+     * @param  {string} "10-732" ProjectId
+     * @param  {Object} {"customVar": ["init"]} set a first custom variable, e.g,. "init" is setted to customVar1
```
Comment (tsuyukimakoto) [inline src/t/setcustomvar.pug]: setted → set で。

**Comment by tsuyukimakoto** [comment]
[LGTM](http://lgtm.in/i/KTCyaE92q)

[:+1:](http://lgtm.in/u/KTCyaE92q) [:-1:](http://lgtm.in/r/KTCyaE92q)

**Comment by Surgo** [comment]
[LGTM](http://lgtm.in/i/EGdWHe96q)

[:+1:](http://lgtm.in/u/EGdWHe96q) [:-1:](http://lgtm.in/r/EGdWHe96q)

**Pull request merged**